sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -5,7 +5,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.azure</groupId>
   <artifactId>azure-sdk-bom</artifactId>
-  <version>1.0.2</version>
+  <version>1.0.3-beta.1</version>
```

Comment:
For beta, don't we always use minor version increment or for boms we use just the patch version?

Reply:
For BOMs we've planned to use just patches (for now)